2024-08-26 11:42:19,211 | WARNING | Неверный тип данных для объекта Runner
Traceback (most recent call last):
  File "C:\Users\User\PycharmProjects\Логирование\Логирование\.venv\Логирование 12_4.py", line 22, in test_run
    runner_1 = r_w_t.Runner(200, 50)
               ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User\PycharmProjects\Логирование\Логирование\.venv\rt_with_exceptions.py", line 6, in __init__
    raise TypeError(f'Имя может быть только строкой ---> class str, передано {type(name).__name__}')
TypeError: Имя может быть только строкой ---> class str, передано int
2024-08-26 11:42:19,214 | WARNING | Скорость для Runner не верна
Traceback (most recent call last):
  File "C:\Users\User\PycharmProjects\Логирование\Логирование\.venv\Логирование 12_4.py", line 11, in test_walk
    runner_1 = r_w_t.Runner('Mikhail', -30)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User\PycharmProjects\Логирование\Логирование\.venv\rt_with_exceptions.py", line 11, in __init__
    raise ValueError(f'Скорость не может быть отрицательной, сейчас {speed}')
ValueError: Скорость не может быть отрицательной, сейчас -30
